vidarrtest-regression.json.in

```diff
@@ -10,7 +10,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/smallBamQc/expected_output/singleTS.calculate.output",
+                "output_metrics": "/.mounts/labs/gsi/testdata/smallBamQc/output_metrics/workflow_test_01.metrics",
```

Comment:
I would avoid changing names for output metrics files - they are helpful when we do development and do not expose any sensitive or OICR-specific information.